sections/04_study.md

```diff
@@ -264,121 +264,236 @@ and [#258](https://github.com/greenelab/deep-review/issues/258).`
+universal enhancer sequence signal or marker for enhancers, and some literature
+suggests that enhancers and promoters may be just categories along a spectrum
+[@doi:10.1016/j.tig.2015.05.007]. One study [@doi:10.1101/gr.173518.114.] even
+showed that only 33% of predicted regulatory regions could be validated, while a
+class of "weak" predicted enhancers were strong drivers of expression.
+classified as positive in the experiment Yet there is growing evidence for their
+vast ubiquity, making them possibly the predominant functional non-coding
+element. Thus, identifying enhancers is critical yet the the search space is
+embarrassingly large.
+
+While prior (non-deep learning) approaches have made steady improvements on
+promoter prediction [@doi:10.1101/gr.7.9.861], there is little consensus on the
+best approach and performance is poor. Typically algorithms will recognize only
+half of all promoters, with a accompanying high false positive rate
+[@doi:10.1101/gr.7.9.861]. Methods with better sensitivity generally do so at
+the cost of poorer specificity. Conventional identification of enhancers has
+leaned heavily on simple conservation or laborious experimental techniques, with
+only moderate sensitivity and specificity. For example, while chromatin
+accessibility has often been used for identifying enhancers, this also
+"recognizes" a wide variety of other functional elements, like promoters,
+silencers and repressors.
+
+The complex nature of CREs (and our ignorance at to what are the important
+features of them) therefore seems a good subject for deep learning approaches.
+Indeed, neural networks were used for promoter recognition as early as 1996,
+albeit with mixed results [@doi:0.1016/S0097-8485(96)80015-5]. Since then, there
+have been much work in applying deep learning to this area, although little in
+the way of comparative studies or reviews. We will therefore focus on a few
+recent characteristic studies to outline the state of the art and extant
+problems.
+
+Most broadly, Kelley et al. [@doi:10.1101/gr.200535.115] trained CNNs on DNA
+accessibility datasets, getting an marked improvement on previous methods,
+albeit still with a high false positive rate. (Note as above, using DNA
+accessibility conflates enhancers with other functional sites.) This study also
+featured a useful interpetability approach, introducing known protein binding
+motifs into sequences and measuring the change in predicted accessibility.
+
+Umarov et al. [@doi:10.1371/journal.pone.0171410] demonstrated the use of
+Convolutional Neural Networks in recognizing promoter sequences, achieving
+markedly better performance than conventional methods (sensitivity and
+specificity exceeding 90%). While some results were achieved over bacterial
+promoters (which are considerably simpler in structure), surprisingly roughly
+similar performance was found for human promoters. This work also included a
```

Comment:
My take is that it's not quite the same but certainly analogous. Good catch, I'll put in a ref.